### ISSUE TITLE
Unlocking construction rights using "Sandbox Mode" will cost money

### ISSUE THREAD

**Issue by karst** [OpenRCT2/OpenRCT2#21167]
### Operating System

Windows 10, 64-Bit

### OpenRCT2 build

OpenRCT2, v0.4.7-16-g872b28c (872b28c on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2, RollerCoaster Tycoon Classic

### Describe the issue

When unlocking new land using the "Sandbox mode" cheat, unlocking owned land or construction rights will cost money, as if you were using the buy land tool (which it isn't).

It's very well possible this was the intention when it was implemented, but it  might as well be an oversight. It seems very odd for a cheat to cost money.

### Steps to reproduce

- Enable Sandbox Mode in cheats.
- Disable "No Money" if the No Money cheat is on.
- Go to the park map
Claim land that isn't part of the park, you will see you loose money for this.

**Comment by ZehMatt** [comment]
I don't think that Sandbox means the same as No Money, it just lifts all restrictions which a scenario might have.

**Comment by karst** [comment]
It acts as if you are buying land, while this isn't really the case. This function claims land

**Comment by HtotheTML** [comment]
You're talking about when you're setting the park boundaries with the sandbox-specific land rights tools, yeah? Those shouldn't cost money, since it's different from the normal land rights tool, and only available if sandbox mode is on.
